types/entities.ts

```diff
@@ -38,6 +38,33 @@ export interface User {
   bioHtml: string | null;
 }
 
+export interface SeriesWorkSummary
+  extends Omit<
+    WorkSummary,
+    "category" | "publishedAt" | "rating" | "tags" | "stats" | "locked"
+  > {
+  tags: Omit<WorkSummary["tags"], "warnings">;
+  stats: Omit<WorkSummary["stats"], "comments">;
+}
+
+export interface Series {
+  id: string;
+  name: string;
+  begunAt: string;
```

Comment:
I would swap this with `startedAt`, since "begun" is less common in code and an irregular verb

Reply:
Sounds good!